modules/ip/datapath/ip_input.c

```diff
@@ -30,6 +30,28 @@ enum edges {
 	EDGE_COUNT,
 };
 
+#ifdef __GROUT_UNIT_TEST__
+#include <gr_cmocka.h>
+
+// The functions are defined as static inline in the original code, which means they are
+// expanded at compile-time and do not have linkable symbols. This makes them impossible
+// to wrap directly using CMocka's function wrapping mechanism.
+//
+// To work around this limitation, we use `#define` to replace calls to these functions
+// with our own wrapped implementations. This ensures that all occurrences of the
+// functions in the tested code are redirected to the wrapped versions during unit testing.
+static void __wrap_rte_node_enqueue_x1(
+	struct rte_graph *graph,
+	struct rte_node *node,
+	rte_edge_t next,
+	void *obj
+);
```

Comment:
For now we only have one single unit test for datapath code. But thinking ahead, I wonder if we should move this function definition into `gr_graph.h` and force all code to include it instead of `rte_graph{_worker}.h`.

Is `static` required on a declaration? It feels weird.